libs/sdk/src/rhp.spec.ts

```diff
@@ -123,16 +125,17 @@ describe('rhp', () => {
         const sdk = await initSDKTest()
         const json: RPCReadSectorResponse = {
           proof: [
-            'h:457256d6a1603bef7fa957a70b5ba96a9def2fea8b4c1483060d7ba5cf8a072c',
+            '457256d6a1603bef7fa957a70b5ba96a9def2fea8b4c1483060d7ba5cf8a072c',
           ],
-          sector: 'AQID',
+          // sector: 'AQID',
```

Comment:
@n8maninger latest types no longer have `sector`, how is the data passed? 🤔

Reply:
It's read manually after reading the response data to enable streaming. You read `dataLength` bytes from the stream.